npm install fails with multiple compilation errors

Many node-gyp compilation errors when installing with npm finally resulting in an msbuild exit code:1 

```
##redacted long list of similar errors

          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(317) : see declaration of 'v8::Handle<v8::Value
  >::Handle'
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(52): error C2039: 'New' : is not a member of 'v8::String' [C:\Users\Jonny\AppData\Roaming\npm\no de_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt.vcxproj]
          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(52): error C3861: 'New': identifier not found [C:\Users\Jonny\AppData\Roaming\npm\node_modules\s wagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt.vcxproj]
          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(1599) : see declaration of 'v8::String'
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(68): error C2248: 'v8::HandleScope::HandleScope' : cannot access protected member declared in cl ass 'v8::HandleScope' [C:\Users\Jonny\AppData\Roaming\npm\node_modules\swagger2blueprint\node_modules\fury\nod e_modules\robotskirt\build\robotskirt.vcxproj]
          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(816) : see declaration of 'v8::HandleScope::Han
  dleScope'
          C:\Users\Jonny\.node-gyp\0.12.6\deps\v8\include\v8.h(800) : see declaration of 'v8::HandleScope'
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(68): error C3083: 'ObjectWrap': the symbol to the left of a '::' must be a type [C:\Users\Jonny\ AppData\Roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt. vcxproj]
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(68): error C2039: 'Unwrap' : is not a member of 'node' [C:\Users\Jonny\AppData\Roaming\npm\node_ modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt.vcxproj]
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(68): error C2065: 'Unwrap' : undeclared identifier [C:\Users\Jonny\AppData\Roaming\npm\node_modu les\swagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt.vcxproj]
c:\users\jonny\appdata\roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\sr c\version.hpp(68): fatal error C1003: error count exceeds 100; stopping compilation [C:\Users\Jonny\AppData\Ro aming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt\build\robotskirt.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\12.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Jonny\AppData\Roaming\npm\node_modules\swagger2blueprint\node_modules\fury\node_modules\robotskirt
gyp ERR! node -v v0.12.6
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok
```

npm version: 2.11.2

I think this may be expected with Node 0.12 since the APIs are no longer compatible with those of 0.10.

Some research indicates that Robotskirt is now deprecated as Sundown is dead, and it has several security bugs.

Can you advise please.

--- a/index.js
+++ b/index.js
@@ -19,15 +19,15 @@ var parser = require('yargs')
   .strict();
 
 fury.use(require('fury-adapter-apib-serializer'));
+fury.use(require('fury-adapter-swagger'));
 
 // Take a loaded Swagger object and converts it to API Blueprint
 function convert(swagger, done) {
-  fury.parse({source: swagger}, function (parseErr, api) {
+  fury.parse({source: swagger}, function (parseErr, res) {
     if (parseErr) {
       return done(parseErr);
     }
-
-    fury.serialize({api: api}, function (serializeErr, blueprint) {
+    fury.serialize({api: res.api}, function (serializeErr, blueprint) {
       if (serializeErr) {
         return done(serializeErr);
       }

--- a/package.json
+++ b/package.json
@@ -26,8 +26,9 @@
     "mocha": "^2.2.5"
   },
   "dependencies": {
-    "fury": "^0.9.0",
-    "fury-adapter-apib-serializer": "^0.1.0",
+    "fury": "^1.0.3",
+    "fury-adapter-apib-serializer": "^0.1.2",
+    "fury-adapter-swagger": "^0.6.2",
     "request": "^2.57.0",
     "yargs": "^3.10.0"
   }

